_includes/homepage/infocols.html

```diff
@@ -0,0 +1,79 @@
+{%- comment -%}
+  include: 
+    subtitle // optional
+    title
+    url // optional
+    linktext // optional
+    infoboxes[]:
+      title
+      description // optional
+{%- endcomment -%}
+<section class="px-8 px-sm-16 px-md-24 py-24 {{ gray_container_class }}">
+  <div class="infocols-container">
+    <div class="infocols-section is-flex">
+      <div class="infocols-header is-flex">
+        {%- if include.subtitle -%}
+          <p class="subtitle-2">
+              {{ include.subtitle }}
+          </p>
+        {%- endif -%}
+
+        <h1 class="h1 has-text-secondary has-text-centered">
+          {{ include.title }}
+        </h1>
+      </div>
+
+      <div class="infocols-infobox-section-grid is-flex">
```

Comment:
this whole chunk is abit difficult to follow due to the layers of nesting involved. would it better structured using `includes`?

Reply:
done!